packages/kit/src/resolve.ts

```diff
@@ -150,17 +194,35 @@ export async function resolveNuxtModule (base: string, paths: string[]) {
 // --- Internal ---
 
 async function existsSensitive (path: string) {
-  if (!existsSync(path)) { return false }
-  const dirFiles = await fsp.readdir(dirname(path))
-  return dirFiles.includes(basename(path))
+  if (!existsSync(path)) {
+    return false
+  }
+
+  const directoryFiles = await fsp.readdir(dirname(path))
+
+  return directoryFiles.includes(basename(path))
 }
 
 // Usage note: We assume path existence is already ensured
 async function isDirectory (path: string) {
-  return (await fsp.lstat(path)).isDirectory()
+  const pathStats = await fsp.lstat(path)
+
+  return pathStats.isDirectory()
 }
 
-export async function resolveFiles (path: string, pattern: string | string[], opts: { followSymbolicLinks?: boolean } = {}) {
-  const files = await globby(pattern, { cwd: path, followSymbolicLinks: opts.followSymbolicLinks ?? true })
-  return files.map(p => resolve(path, p)).filter(p => !isIgnored(p)).sort()
+export async function resolveFiles (
+  path: string,
+  pattern: string | string[],
+  options: { followSymbolicLinks?: boolean } = {}
+) {
```

Comment:
can we revert the lint change that splits function parameters into separate lines? 🙏